patient-search/api/src/main/java/gov/cdc/nbs/entity/elasticsearch/ElasticsearchPerson.java

```diff
@@ -369,4 +370,19 @@ public class ElasticsearchPerson {
 
     @Field(name = "sex_unk_reason_cd", type = FieldType.Keyword)
     private String sexUnkReasonCd;
+
+    @Field( type = FieldType.Nested)
+    private List<NestedName> name;
+
+    @Field( type = FieldType.Nested)
+    private List<NestedAddress> address;
+
+    @Field( type = FieldType.Nested)
+    private List<NestedPhone> phone;
+
+    @Field( type = FieldType.Nested)
+    private List<NestedEmail> email;
+
+    @Field( type = FieldType.Nested)
+    private List<NestedRace> race;
```

Comment:
very nit-picky but can we do plural field variables for these lists? E.G: names instead of name.

Reply:
I don't think it's nit-picky. It's a great point and that's originally how I coded it.  However I noticed the nested elasticsearch queries of the form address.streetAddress1 in the docs so it was a lazy way of not having to add a mapping.  I don't have strong feelings either way.  Your thoughts?

Reply:
I'm fine with leaving it as is if thats the elasticsearch naming convention.